### PR TITLE
test: fill form and challenges

### PR DESCRIPTION
**Pull request by Rendxn**

### CODE REVIEW

File: cypress/page/personal-form.page.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  }
+
+  public submitForm(data: IPersonalFormData): void {
+    cy.get(this.firstNameInput).type(data.firstName);
+    cy.get(this.lastNameInput).type(data.lastName);
+    cy.get(this.emailInput).type(data.email);
+
+    // Since the ::before pseudo-element of the label for this radio
+    // is "covering" the input, the user (Cypress)
+    // "can't see it", therefore `force` is needed.
```
Comment (manuelq12) [inline cypress/page/personal-form.page.ts:44]: Please erase this comments

File: cypress/page/personal-form.page.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    cy.get(this.genderInput)
+      .filter(`[value="${data.gender}"]`)
+      .check({ force: true });
+
+    // {selectall} is necessary, because .clear() will
+    // make the page crash...
```
Comment (manuelq12) [inline cypress/page/personal-form.page.ts:50]: These as well

File: cypress/page/personal-form.page.ts [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        .check({ force: true });
+    });
+
+    cy.get(this.mobileInput).type(data.mobileNumber);
+    cy.get(this.addressInput).type(data.currentAddress);
+    cy.get(this.stateInput).type(data.state, { force: true }).type("{enter}");
+    cy.get(this.cityInput).type(data.city, { force: true }).type("{enter}");
+
+    // The button is covered by the footer when the viewport
+    // is not high enough.
```
Comment (manuelq12) [inline cypress/page/personal-form.page.ts:65]: Erase comments

File: cypress/page/personal-form.page.ts [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    cy.get(this.submitButton).type("{enter}", { force: true });
+  }
+
+  public validateItems(
+    expectedMessage: string,
+    expectedValues: IPersonalFormData
+  ) {
+    // Parse and format the date :pain:
```
Comment (manuelq12) [inline cypress/page/personal-form.page.ts:73]: Erase this comments

(:PAIN: HAHAHA)

**Comment by Rendxn** [comment]
Re-ran jobs and they passed this time.

**Review by holgiosalos — APPROVED** [review]
good job!

**Pull request merged**